mitodl/hugo-course-publisher
# updated markdown - code blocks and left nav

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/hugo-course-publisher/issues/181 and https://github.com/mitodl/hugo-course-publisher/issues/154

#### What's this PR do?
Updates sample markdown with the most recent changes from fixing the above bugs.

#### How should this be manually tested?
Run the site or visit the deploy preview, verify that the issues in the bug reports above are resolved.
